toolkit/tools/scheduler/scheduler.go

```diff
@@ -346,6 +366,8 @@ func buildAllNodes(stopOnFailure, canUseCache bool, packagesToRebuild, testsToRe
 		if len(buildState.ActiveBuilds()) == 0 && len(channels.Results) == 0 {
 			if useCachedImplicit {
 				err = fmt.Errorf("could not build all packages")
+				// Temporarily print debug information about the stuck node.
```

Comment:
I don't think it's temporary - it's useful info.

Reply:
This will generate a HUGE amount of output for a "normal" failed build (I could see thousands of lines easily enough) We already print out details about the stuck RPMs if everything is working as intended.